splash/lua_runner.py

```diff
@@ -102,6 +106,9 @@ def dispatch(self, cmd_id, *args):
             try:
                 args = args or None
 
+                if self._is_stopped:
+                    raise StopIteration
```

Comment:
self.result cannot be anything except an empty string, so calling it is harmless